ZenPacks/zenoss/Import4/model.py

```diff
@@ -139,15 +139,24 @@ def zenpack(self):
         # zenpack --restore AND --ignore-services and --keep-pack
         log.info('Fixing zenpack in zodb and files on the image ...')
         _cmd = "zenpack --restore --keep-pack=ZenPacks.zenoss.Import4"
-        self.exec_cmd(_cmd)
+        self.exec_cmd(_cmd, status_key='numZenPacks', status_re='looking for', status_max=self.zenpack_count,
+                      to_log_re='looking|loading|Previous')
 
         log.info("zenpacks restored:%s", codeString[ExitCode.SUCCESS])
+
+        # do a quick post validation
+        self.postvalidate()
         return
 
     #==========================================================================
 
     def postvalidate(self):
-        self.__NOT_YET__()
+        log.info("validating the zenpacks and devices")
+        _cmd = "zendmd --script=%s/check_model.dmd" % self.binpath
```

Comment:
```check_model.dmd``` should be moved into here.  You can use soemthing like this to get a handle to dmd:
```
        import Globals
        from Products.ZenUtils.Utils import unused
        unused(Globals)
        from Products.ZenUtils.ZenScriptBase import ZenScriptBase

        dmd = ZenScriptBase(noopts=True, connect=True).dmd
```